shopinvader_validation_token/tests/common.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from contextlib import contextmanager
+
+from odoo.addons.shopinvader.tests.common import CommonCase
+
+
+class CommonValidationToken(CommonCase):
+    def _get_usage(self):
+        return ""
+
+    def _get_partner(self):
+        return self.env["res.partner"].browse()
+
+    def _get_notification_type(self):
+        return ""
+
+    def setUp(self, *args, **kwargs):
```

Comment:
you can do everything in setUpClass

Reply:
@acsonefho Could you attend ?

@simahawk Not the work_on_services part I think